### PR TITLE
enhance upgrade cluster doc

### PR DESCRIPTION
**Pull request by KubeKyrie**
优化文档：工作集群离线升级指南

### CODE REVIEW

File: docs/zh/docs/kpanda/best-practice/update-offline-cluster.md [hunk 1]
```diff
@@ -66,28 +59,27 @@
...
-        │   ├── files
-        │   │   ├── import_files.sh
-        │   │   └── offline-files.tar.gz #二进制文件
-        │   └── images
-        │       ├── import_images.sh
-        │       └── offline-images.tar.gz #镜像文件
-        └── kubeanofflineversion.cr.patch.yaml
+   data
+   ├── amd64
+   │   ├── files
```
Comment (windsonsea) [inline docs/zh/docs/kpanda/best-practice/update-offline-cluster.md:74]:
```suggestion
    data
    ├── amd64
    │   ├── files
```
这些缩进不要修改吧

**Comment by windsonsea** [comment]
See preview:
https://deploy-preview-5638--daocloud-docs.netlify.app/kpanda/best-practice/update-offline-cluster

改了缩进，那部分就乱了

**Review by windsonsea — APPROVED** [review]
Thanks. Everything is ok now.

**Comment by KubeKyrie** [comment]
感谢海峰哥帮忙修改

**Pull request merged**